lib/rules/no-else-return.js

```diff
@@ -223,7 +223,7 @@ module.exports = {
                     return;
                 }
                 consequents.push(node.consequent);
-                alternate = node.alternate;
+                alternate = node.alterante;
```

Comment:
I think the build is failing because you misspelled `alternate` here (and also in the corresponding line below).